src/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+.form-wrapper {
+  width: 100%;
+  display: flex;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  padding: 0;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  margin: 20px 0;
+  height: 70px;
+}
+
+.form-wrapper:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-wrapper--small {
+  height: 42px;
+}
+
+.search {
+  width: 100%;
+  border: 0;
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+}
+
+.search-wrapper--small {
```

Comment:
Change the selector from `.search-wrapper--small` to `.search--small` to target the smaller search bar and set the font size.